mkdocs.yml

```diff
@@ -72,6 +72,8 @@ nav:
       - Using AzureTRE for Research:
           - Introduction: using-tre/tre-for-research/index.md
           - Set up of a Virtual Machine: using-tre/tre-for-research/using-vms.md
+      - Using AzureTRE as an Airlock Manager:
+          - Reviewing Airlock Requests: using-tre/tre-for-airlock-reviewer/review-airlock-request.md
```

Comment:
I think that should be another doc in the **Research** category.

Reply:
My thinking was to separate this section by roles

Reply:
We anyone have very few docs and a group with just 1 doc isn't needed IMO. It's not like the researcher group has much info anyone.
I'd opt for a simplified approach in the docs as well.

Reply:
We do have more docs coming in the researcher group as James is working on adding docs for using Airlock for researchers: https://github.com/microsoft/AzureTRE/issues/2783 
Also if we keep the structure then existing links won't break.

The new Reviewer group isn't likely to get more docs though I think, so I could move it to tre-for-research folder. The only reason I don't do it is because I wanted to have roles separation, as I think customer was also saying that it would be useful to have docs for researchers in a single place.

Reply:
I don't think we need the `Using AzureTRE as an Airlock Manager` heading. You call out to do reviews you need to be an airlock manager at the top of the doc.

It might be in time we need to split it down, but not sure needed at the moment.

Reply:
@marrobi  Fair enough, so do you think it should go to "Using TRE for research" section? If anything goes into this section, do we need it at all?

Reply:
I've moved the new file into "Using TRE for research", @marrobi  PTAL